src/Select.js

```diff
@@ -1063,23 +1111,23 @@ export default class Select extends Component<Props, State> {
   // ==============================
   // Renderers
   // ==============================
-
-  renderScreenReaderStatus() {
-    const { screenReaderStatus } = this.props;
-    return (
-      <A11yText aria-atomic="true" aria-live="polite" role="status">
-        {screenReaderStatus({ count: this.countOptions() })}
-      </A11yText>
-    );
+  constructAriaLiveMessage () {
```

Comment:
each invocation of constructAriaLiveMessage creates an aria live message to represent the following: 
- focusedValue 
- focusedOption 
- results (amount of options available) and inputvalue 
- ariaLiveContext

We construct this every time on render, in order for us to not have to imperatively infer value focus or option focus change in event calls or within componentWillReceiveProps. valueFocus and optionFocus can change due to any number of user interactions and non user interaction based side-effects from async loading to controlled prop changes outside of our intended scope. Likewise with results.